examples/complete/aws_login_rolesanywhere.sh

```diff
@@ -101,11 +103,14 @@ cred=$(aws_signing_helper \
   --cert-selector "Key=x509Serial,Value=$PIV_CERT_SERIAL" \
   --trust-anchor-arn "$TRUST_ANCHOR_ARN" \
   --profile-arn "$PROFILE_ARN" \
-  --role-arn "$ROLE_ARN")
+  --role-arn "$ROLE_ARN" \ 
+  --session-duration 3599)
```

Comment:
in my testing session duration wasn't needed. It defaults to a one hour, 3600s session token